src/components/Common/FilePreviewDialog.tsx

```
@@ -433,99 +433,66 @@
-                        t("rotate_left"),
-                        "l-corner-up-left",
-                        () => handleRotate(-90),
-                        false,
-                      ],
-                      [
-                        t("rotate_right"),
-                        "l-corner-up-right",
-                        () => handleRotate(90),
-                        false,
-                      ],
+                    {[ 
```

ESLint

[error] Delete `·`

(prettier/prettier)